keras/src/backend/openvino/excluded_concrete_tests.txt

```diff
@@ -9,6 +9,7 @@ NumpyDtypeTest::test_any
 NumpyDtypeTest::test_argpartition
 NumpyDtypeTest::test_array
 NumpyDtypeTest::test_bartlett
+NumpyDtypeTest::test_blackman
```

Comment:
Please also add the function as raising `NotImplementedError` in the OpenVino `numpy.py`